lib/sql.js

```diff
@@ -1185,8 +1185,22 @@ SQLConnector.prototype.buildOrderBy = function(model, order) {
     return '';
   }
   var self = this;
-  if (typeof order === 'string') {
-    order = [order];
+  var usedConnector = this.dataSource.connector.name;
+
+  if (order === 'rand') {
```

Comment:
Is it possible for the name `rand` to clash with other valid order values, e.g. when a user has a database with a column called `rand`? Should we use a different value here, e.g. `$rand`?